include/SFML/Graphics/Shape.hpp

```diff
@@ -181,6 +199,16 @@ class SFML_GRAPHICS_API Shape : public Drawable, public Transformable
     ////////////////////////////////////////////////////////////
     float getOutlineThickness() const;
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Get the limit on the ratio between miter length and outline thickness
+    ///
+    /// \return Limit on the ratio between miter length and outline thickness
```

Comment:
This documentation could need a bit more explanation on what "miter" is. If I don't know what "miter" is, the documentation essentially explains things by self reference as one also wouldn't understand what "miter length" is.

Reply:
I tried to make is clearer adding documentation to `setMiterLimit` method. Feedback is welcome.